test/ibmq/runtime/test_runtime_integration.py

```diff
@@ -149,6 +150,20 @@ def test_upload_program(self):
         self.assertTrue(program)
         self.assertEqual(max_execution_time, program.max_execution_time)
 
+    def test_set_visibility(self):
+        """Test setting the visibility of a program."""
+        program_id = self._upload_program()
+        # Get the initial visibility
+        prog: RuntimeProgram = self.provider.runtime.program(program_id)
+        start_vis = prog.to_dict()['is_public']
```

Comment:
Why not just do `prog.is_public()`? Instead of hard coding the key.

Reply:
Fixed! in-place.